client/src/common/components/CardItem.jsx

```diff
@@ -1,21 +1,34 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
+import { PlusIcon } from 'assets/images/icons';
+
 export const CardColorType = {
   BROWN: 'card/BROWN',
   ORANGE: 'card/ORANGE'
 };
 
-const CardItem = ({ color, description, name }) => (
+const CardItem = ({ color, description, name, withPlus }) => (
   <div
     className={classNames('card-item', {
       'card-item--orange': color === CardColorType.ORANGE,
-      'card-item--brown': color === CardColorType.BROWN
+      'card-item--brown': color === CardColorType.BROWN,
+      'card-item--gray': withPlus,
+      'card-item__plus': withPlus
     })}
   >
-    <h3 className="card-item__heading">{name}</h3>
-    <p className="card-item__description">{description}</p>
+    {withPlus ? (
```

Comment:
when preparing this task I have tried to use just `<button>` and add it where need it. But this solutions seems to be not scalable and not intuitive for me. This element looks like CardItem, so I decided to use CardItem with additional withPlus props to generate gray CardItem instead of colorful one.
@marekrozmus what do you think? Should I separate this to indpenedent component?

Reply:
Please create separate component.